modules/ROOT/pages/create-inbound-message-flow.adoc

```diff
@@ -1,119 +1,136 @@
+The source message type in an inbound message flow defines the structure of the messages that Partner Manager receives from a partner or 3rd party connection. Partner Manager supports the EDIFACT and X12 formats for source message types in an inbound message flow.
 
-[source-message-type]
-== Configure the Source Message Type
+To configure the source message type:
 
 . In the message flow, expand the *Source message* section and click *Select*.
-. In the *Select message type* window:
-.. To use an existing message type, select the message type from the list and then click *Select*.
-.. To create a new message type, click *New* next to the search box, configure the X12 or EDIFACT xref:partner-manager-create-message-type.adoc[message type], and click *Save*.
-The message type and its validation settings are added to the message flow.
-. If your partner expects your organization to return functional acknowledgements:
-.. In the *Acknowledgement* section, select the check box next to *Send 997 functional acknowledgment (X12)* or *Send CONTRL functional acknowledgement (EDIFACT)* .
-.. Click *Select*.
-.. In the *Select an endpoint: Send acknowledgment to <partner>* window, click *Select* to specify the endpoint from which to receive the acknowledgment transactions from your partner.
+. In the *Select message type* window, do either of the following:
+* Select an existing message type and click *Select*.
+* Click *New* to create a new receive from partners type, as described in <<xref:partner-manager-create-message-type.adoc,Create and Configure Message Types>>. *Vijay, is receive from partners correct in the last sentence?.*
+
 +
-.. Alternatively, you can create a new send to partner endpoint:
-... Click *New*.
-... In the *Owner* field, select the endpoint owner.
-... In the *Protocol* field, select and configure the protocol.
-... Click *Save*.
+The message type and its validation settings are added to the message flow.
+
+If the source message type is X12 or EDIFACT and your partner expects your organization to return functional acknowledgements, perform these additional steps:
+
+. In the *Acknowledgement* section, select the check box next to *Send 997 functional acknowledgment (X12)* or *Send CONTRL functional acknowledgement (EDIFACT)*.
+. Click *Select*.
+. In the *Select an endpoint: Send acknowledgement to <partner>* window, do either of the following:
+* Select the endpoint to receive the acknowledgment messages from your partner and click *Select*.
+* Click *New* to create a new send to endpoint, as described in <<xref:create-endpoint.adoc,Create and Configure Endpoints>>.
+
 +
 The acknowledgment is added to the message flow.
 
 [[import-map]]
-== Import the Translation Map
+=== Import the Translation Map
```

Comment:
This is called a "transform" in the overview topic: can we standardize on one of the two terms throughout?